src/components/v5/frame/ColonyHome/partials/ReputationChart/partials/Chart.tsx

```diff
@@ -0,0 +1,81 @@
+import { type PieTooltipProps, ResponsivePie } from '@nivo/pie';
+import React, { type FC } from 'react';
+import { defineMessages } from 'react-intl';
+
+import Numeral from '~shared/Numeral/Numeral.tsx';
+import { formatText } from '~utils/intl.ts';
+
+import { pieChartConfig } from '../consts.ts';
+import { type ReputationChartDataItem } from '../types.ts';
+
+import Legend from './Legend.tsx';
+import LegendItem from './LegendItem.tsx';
+
+const displayName = 'v5.frame.ColonyHome.ReputationChart.partials.Chart';
+
+const MSG = defineMessages({
+  noReputationLabel: {
+    id: `${displayName}.noReputationLabel`,
+    defaultMessage: 'There is no reputation in the colony yet',
+  },
+});
+
+const EMPTY_CHART_ITEM = {
+  id: 'noReputation',
+  label: formatText(MSG.noReputationLabel),
+  value: 100,
+  color: '--color-gray-200',
+};
+
+interface ChartProps {
+  data: ReputationChartDataItem[];
+}
+
+const TemporaryTooltip: FC<PieTooltipProps<ReputationChartDataItem>> = ({
+  datum: { label, value },
+}) => {
+  return (
+    <div className="bg-base-black p-2 text-base-white">
+      {label}
+      (<Numeral value={value.toFixed(2)} />
+      %)
```

Comment:
I think this is already in the `feat/enhanced-dashboard` component so can be dealt with later but the tooltip copy currently looks like this:

![image](https://github.com/user-attachments/assets/d225f44d-01ff-4b70-8a70-ad4615d56bb6)

There should be a space between `all other` and the opening parenthesis, the font needs to be 10px and font weight at 600. We can also use the `suffix` prop for the `Numeral` component too if you'd like:

```js
<div className="bg-base-black p-2 text-xs font-semibold text-base-white">
  {label} (<Numeral value={value.toFixed(2)} suffix="%" />)
</div>;
```

![image](https://github.com/user-attachments/assets/92b08cea-628f-4da4-a466-b80da96aa853)

Reply:
It's solved here https://github.com/JoinColony/colonyCDapp/pull/3116

Reply:
Nice catch @rumzledz 🙌 
Fixed it in my last commit
<img width="233" alt="image" src="https://github.com/user-attachments/assets/cee49818-0fad-4ca5-9a37-550a4ca18b47">